server/src/server.rs

```diff
@@ -59,12 +70,14 @@ define_result!(Error);
 pub struct Server<C, Q> {
     http_service: Service<C, Q>,
     rpc_services: RpcServices,
+    mysql_service: mysql::MysqlHandler<C, Q>,
```

Comment:
>I think this modification is not very good, so I finally use msql-srv

I'm ok with this. Add trait constraints is acceptable if we need to.

Reply:
Using `tokio` is indeed more better than `threadpool`.😂 I will change him to `tokio` form.